ftw/contacts/profiles/default/workflows.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<object name="portal_workflow" meta_type="Plone Workflow Tool">
+ <bindings>
+  <type type_id="ftw.contacts.ContactFolder"/>
+  <type type_id="ftw.contacts.Contact"/>
```

Comment:
This *removes* the workflows for those types.
I think we want the default-workflow for `ContactFolder` and `Contact` and probably `Member`?
On the other hand, the simplelayout block should not have a workflow and exactly that should be done for the block..